apps/prairielearn/src/migrations/20241007203121_draft_questions.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE questions
+ADD COLUMN IF NOT EXISTS is_draft boolean NOT NULL DEFAULT (false);
+
+ALTER TABLE pl_courses
+ADD COLUMN IF NOT EXISTS draft_number integer NOT NULL DEFAULT (0);
+
+CREATE TABLE IF NOT EXISTS questions_draft_metadata (
+  id bigint NOT NULL PRIMARY KEY,
+  draft_created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  draft_created_by BIGINT REFERENCES users (user_id) ON DELETE SET NULL ON UPDATE CASCADE,
+  draft_updated_by BIGINT REFERENCES users (user_id) ON DELETE SET NULL ON UPDATE CASCADE
```

Comment:
Is a `draft_` prefix necessary given that the table name also includes "draft"?

Reply:
removed prefixes in 562ccc7ca4238b061cc5fed79757979f8c0db35b.